File: CST-210/PolyStack/PolyStack/StackableString.cpp
#include "StackableString.h"

StackableString::StackableString(string str)
{
    m_data = str;
}

string StackableString::GetData(void)
{
    return m_data;
}

bool StackableString::HasData(void)
{
    return true;
}

File: CST-105/PolyStack/src/StackableString.java

public class StackableString extends StackableObject
{
	private String Data;
	
	public StackableString(String data)
	{
		Data = data;
	}
	
	@Override
	public String GetData()
	{
		return Data;
	}

	@Override
    public Boolean HasData()
    {
        return(true);
    }
}
